src/inference/src/system_conf.cpp

```diff
@@ -225,14 +228,18 @@ std::vector<int> get_available_numa_nodes() {
 #        endif
 #    endif
 
-std::vector<std::vector<int>> get_num_available_cpu_cores() {
+std::vector<std::vector<int>> get_proc_type_table() {
     return cpu._proc_type_table;
 }
 
 bool is_cpu_map_available() {
     return cpu._proc_type_table.size() > 0 && cpu._num_threads == cpu._proc_type_table[0][ALL_PROC];
 }
 
+int get_num_numa_nodes() {
+    return cpu._sockets;
```

Comment:
Is numa nodes always equal sockets? Or we need to rename `cpu._sockets` to `cpu._numa_nodes`?

Reply:
ok, done.